test/db/anal/pickle

```diff
@@ -14,13 +14,13 @@ EXPECT=<<EOF
 0x00000003               4d4241  BININT2 0x4142
 0x00000006           4a44434241  BININT 0x41424344
 0x0000000b   954746454443424100  FRAME 0x41424344454647
-0x00000014                 5500  SHORT_BINSTRING ""
-0x00000016         550441424344  SHORT_BINSTRING "ABCD"
-0x0000001c   420400000041424344  BINBYTES "ABCD"
-0x00000025 8d040000000000000041424344  BINUNICODE8 "ABCD"
-0x00000032         4c3334354c0a  LONG "345L"
-0x00000038 63414243440a454647480a  GLOBAL "ABCD EFGH"
-0x00000043         550441424344  SHORT_BINSTRING "ABCD"
+0x00000014                 5500  SHORT_BINSTRING 
+0x00000016         550441424344  SHORT_BINSTRING ABCD
+0x0000001c   420400000041424344  BINBYTES ABCD
+0x00000025 8d040000000000000041424344  BINUNICODE8 ABCD
+0x00000032         4c3334354c0a  LONG 345L
+0x00000038 63414243440a454647480a  GLOBAL ABCD EFGH
```

Comment:
that seems incorrect. its not the same "abcd efgh" than abcd efgh... its two separate arguments. also for the color highlighting its better to have the string starting with `"` imho. if you want you can use one of the ASM_SYNTAX to tweak the output

Reply:
and also add a pseudo plugin that makes all this readable.

Reply:
So there is no real ambiguity if you know how pickle opcodes work but that is a cop out. 
I took the easy way out instead of updating the assembler to use quotes. I will fix it. I will see about adding a pseudo plugin as well.

I will try to amend soon.